analyzer/java/com/engflow/bazel/invocation/analyzer/traceeventformat/CompleteEvent.java

```diff
@@ -102,15 +102,25 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return "CompleteEvent{" +
-        "name='" + name + '\'' +
-        ", category='" + category + '\'' +
-        ", start=" + start +
-        ", duration=" + duration +
-        ", end=" + end +
-        ", threadId=" + threadId +
-        ", processId=" + processId +
-        ", args=" + args +
-        '}';
+    return "CompleteEvent{"
```

Comment:
Can we fix the linter to not do stuff like this? This is _much_ less readable than it was before.

Reply:
Maybe unrelated, but looking at this more closely, why do some of these strings have single quotes?

Reply:
(I mean, I understand, they're single characters, but I'd go with using single-character strings rather than chars.)

Reply:
I've rewritten the code to use `String.format` in #102 - that is hopefully even more readable.